### PR TITLE
Update dev guide with intellij inserting new line configuration

### PR DESCRIPTION
**Pull request by jamiesjc**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ It allows us to do following when saving a file:
 * Remove unnecessary semicolon
 * Add missing @Override annotations
 
+Configure Intellij to [insert a new line at the end of a file](https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file)
```
Comment (HappyRay) [inline CONTRIBUTING.md]: Could you add a heading such as "New line at the end of a file"?

Comment (HappyRay) [inline CONTRIBUTING.md]: Is the link needed?
If need, I try to link to the intellij doc directly if possible.

Comment (jamiesjc) [reply CONTRIBUTING.md]: The link is probably not necessary. I'll remove it.

Comment (jamiesjc) [reply CONTRIBUTING.md]: Sure, I will add the heading.

**Pull request merged**